dist/screeps.d.ts

```diff
@@ -1102,7 +1102,7 @@ interface FindPathOpts {
      */
     maxRooms?: number;
 }
-interface MoveToOpts {
+interface MoveToOpts extends FindPathOpts {
```

Comment:
Acceptable, assuming there are no places without FindPathOpts included in MoveToOpts usage.